Android/app/src/main/res/layout/activity_profile.xml

```diff
@@ -140,6 +140,7 @@
                 android:ellipsize="end"
                 android:focusableInTouchMode="false"
                 android:inputType="textMultiLine"
+                android:maxLength="50"
```

Comment:
is the UI going to be similar as status (I mean, the current count/total count will be shown by the side of edit text?)

Reply:
No, I will add that counter also.

Reply:
Made the suggested changes, added the word counter. @Swati4star can you please review now.